docs/release-notes/mongoid-7.5.txt

```diff
@@ -17,6 +17,12 @@ The complete list of releases is available `on GitHub
 please consult GitHub releases for detailed release notes and JIRA for
 the complete list of issues fixed in each release, including bug fixes.
 
+``Array#update_values`` and ``Hash#update_values`` deprecated
+`````````````````````````````````````````````````````````````
+
+The ``Array#update_values`` and ``Hash#update_values`` methods are deprecated in
+Mongoid 7.5.
```

Comment:
Since we know what the recommended replacements are, can they be included here please?

Reply:
sure